libs/liquidity/src/lib/liquidity-table.tsx

```diff
@@ -440,24 +444,30 @@ export const LiquidityTable = ({
         marryChildren: true,
         children: [
           {
-            headerName: t(`Last time on the book`),
+            headerName: t(`Last time on book`),
             field: 'sla.lastEpochFractionOfTimeOnBook',
             type: 'rightAligned',
-            headerTooltip: t('Last epoch fraction of time on the book.'),
+            headerTooltip: t(
+              'Fraction of time on the book at the end of the last epoch.'
+            ),
             valueFormatter: percentageFormatter,
           },
           {
             headerName: t(`Last fee penalty`),
             field: 'sla.lastEpochFeePenalty',
             type: 'rightAligned',
-            headerTooltip: t('Last epoch fee penalty.'),
+            headerTooltip: t(
+              'Penalty applied on the fees a liquidity provider collected in the last epoch. This percentage increased if an LP did not meet the SLA, or if they met it but other LPs outscored them in the previous epoch.'
```

Comment:
Non blocking: But I think some of these tooltips might be a bit easier to manager with shorter keys.